coronavirus/coronavirus.py

```diff
@@ -823,7 +824,7 @@ def align_sets_at(v0, df):
 
     return res
 
-def get_compare_data(countrynames, rolling=7):
+def get_compare_data(countrynames, rolling=7, regions=False):
```

Comment:
Can you extend the documentation to say what the new variable `regions` is expected to be?